src/features/generic_events/aggregate/index.js

```diff
@@ -99,8 +133,9 @@ export class Aggregate extends AggregateBase {
     const defaultEventAttributes = {
       /** should be overridden by the event-specific attributes, but just in case -- set it to now() */
       timestamp: Math.floor(this.#agentRuntime.timeKeeper.correctRelativeTimestamp(now())),
-      /** all generic events require a pageUrl */
-      pageUrl: cleanURL(getRuntime(this.agentIdentifier).origin)
+      /** all generic events require pageUrl(s) */
+      pageUrl: cleanURL('' + initialLocation),
+      currentUrl: cleanURL('' + location)
```

Comment:
I just want to point that that this (among other changes) were not part of the CDD, and the actual feature itself hasn't been released yet. Noticing a pattern where this UA just keeps growing in size without any customer data or driven by feedback.
Can resolve this comment once read.